source/win32_tinyengine.c

```diff
@@ -5,6 +5,234 @@
 
 #include "tinyengine_types.h"
 
+typedef struct win32_digital_button win32_digital_button;
+struct win32_digital_button
+{
+    u32 Up;
+    u32 Down;
+    b32 Pressed;
+    b32 Released;
+
+    b32 WMDown;
```

Comment:
IsDown?

Reply:
See the above post -- `WMDown` is a (poor) attempt at solving the `WM_INPUT` dilemma